tests/e2e/conftest.py

```diff
@@ -198,48 +190,6 @@ def neuro_login(pip_install_neuromation: None) -> t.Iterator[None]:
             log.info("Deleted")
 
 
-# == generic helpers ==
```

Comment:
these three methods need to be moved to `utils.py` so that we have access to them from `configuration.py`